shared/actions/chat/index.js

```diff
@@ -346,61 +346,94 @@ function * _loadMoreMessages (action: Constants.LoadMoreMessages): SagaGenerator
     next = oldConversationState.get('paginationNext', undefined)
   }
 
-  yield put({type: 'chat:loadingMessages', payload: {conversationIDKey}})
+  yield put({payload: {conversationIDKey, isRequesting: true}, type: 'chat:loadingMessages'})
```

Comment:
so i went and expanded all the chat code to use Creators.*. This actually caught a bunch of stufff ( :) ) but the next commit is going to be larger and maybe introduce a bug so we should retest ( :( ). But ultimately i think we should only use creators as we have too much untyped stuff being dispatched and the typing syntax suxxors